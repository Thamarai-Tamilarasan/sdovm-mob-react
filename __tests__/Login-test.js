import 'react-native';
import React from 'react';
import Login from '../app/components/standalone/login/';
import renderer from 'react-test-renderer';

test('Login SnapShot ', () => {
  const login = renderer.create(<Login />).toJSON();

  expect(login).toMatchSnapshot();
});
