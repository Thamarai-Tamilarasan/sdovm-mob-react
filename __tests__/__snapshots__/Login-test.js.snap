// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login SnapShot  1`] = `
<View
  style={
    Object {
      "backgroundColor": "#E5E5E5",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "space-between",
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <Image
        source={
          Object {
            "testUri": "../../../assets/images/logo.png",
          }
        }
        style={Object {}}
      />
    </View>
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#00529F",
            "fontFamily": "Poppins-SemiBold",
            "fontSize": 28,
            "letterSpacing": -0.24,
          }
        }
      >
        Whereâ€™s My Order?
      </Text>
    </View>
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#009FE0",
              "fontFamily": "Poppins-SemiBold",
              "fontSize": 18,
              "letterSpacing": -0.24,
            }
          }
        >
          Login
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Object {
        "flex": 1,
      }
    }
  />
</View>
`;
